ui/lib/core/addon/helpers/is-active-route.js

```diff
@@ -19,7 +19,7 @@ export default Helper.extend({
     const currentRoute = router.get('currentRouteName');
     let currentURL = router.get('currentURL');
     // if we have any query params we want to discard them
-    currentURL = currentURL.split('?')[0];
+    currentURL = currentURL?.split('?')[0];
```

Comment:
this error's out in our tests. Happens because this is the first time SecretListHeader is being using in an engine, and an engine has different URLs and routes passed into this helper than it does outside an engine.